### PR TITLE
Exposing some model property in BOXComments related requests.

### PR DESCRIPTION
**Pull request by scottliubox**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @scottliubox has signed the CLA. Thanks for the pull request!

File: BoxContentSDK/BoxContentSDK/Requests/BOXFileCommentsRequest.h [hunk 1]
```diff
@@ -10,6 +10,7 @@
 @property (nonatomic, readwrite, assign) BOOL requestAllItemFields;
 @property (nonatomic, readwrite, strong) NSURL *sharedLinkURL;
 @property (nonatomic, readwrite, strong) NSString *sharedLinkPassword; // Only required if the shared link is password-protected
+@property (nonatomic, readonly, strong) NSString *fileID;
```
Comment (RicoYao) [inline BoxContentSDK/BoxContentSDK/Requests/BOXFileCommentsRequest.h:13]: remove property declaration from the .m ?

File: BoxContentSDK/BoxContentSDK/Requests/BOXCommentRequest.h [hunk 2]
```diff
@@ -10,6 +10,7 @@
 @property (nonatomic, readwrite, assign) BOOL requestAllItemFields;
 @property (nonatomic, readwrite, strong) NSURL *sharedLinkURL;
 @property (nonatomic, readwrite, strong) NSString *sharedLinkPassword; // Only required if the shared link is password-protected
+@property (nonatomic, readonly, strong) NSString *commentID;
```
Comment (RicoYao) [inline BoxContentSDK/BoxContentSDK/Requests/BOXCommentRequest.h:13]: remove property declaration from the .m ?

File: BoxContentSDK/BoxContentSDK/Requests/BOXBookmarkCommentsRequest.h [hunk 3]
```diff
@@ -7,6 +7,8 @@
 
 @interface BOXBookmarkCommentsRequest : BOXRequestWithSharedLinkHeader
 
+@property (nonatomic, readonly, strong) NSString *bookmarkID;
```
Comment (RicoYao) [inline BoxContentSDK/BoxContentSDK/Requests/BOXBookmarkCommentsRequest.h:10]: remove property declaration from the .m ?

**Comment by RicoYao** [comment]
+2

File: BoxContentSDK/BoxContentSDK/Requests/BOXFileCommentsRequest.h (hunk 1 above)
Comment (scottliubox) [inline BoxContentSDK/BoxContentSDK/Requests/BOXFileCommentsRequest.h:13]: Good call, however let's remain consistent with the rest of the project right now, which is still adhering to this pattern. We should probably do a round of clean house afterwards.

**Pull request merged**